mobilelib.php

```diff
@@ -156,6 +156,32 @@ function build_allowed_tag_users($userarray) {
     return $returnarray;
 }
 
+/**
+ * @param context_course $context
+ * @param $user
+ * @return bool
+ */
+function check_capability($context, $user)
+{
+    global $result, $USER;
+
+    try {
+        if (has_capability('local/getsmarter:mention_' . $user->shortname, $context)) {
```

Comment:
Could you not just return has_capability('local/getsmarter:mention_' . $user->shortname, $context)? Then you don't need the return false at the end?

Reply:
@PPrinsloo think we can do this here and on the web?

Reply:
Sure, I guess. I normally don't return function results directly in Moodle though, they're not super consistent all over the system.